### PR TITLE
Refining the design for adding another outcome and attaching files

### PR DESCRIPTION
**Pull request by mborsare**
add-outcome-file

### CODE REVIEW

File: app/views/attachments/_fields.html.erb [hunk 1]
```diff
@@ -4,8 +4,10 @@
   <% if f.object.persisted? %>
     <span class="file-name"><%= f.object.name %></span>
   <% else %>
-    <span class="file-name"></span>
+    <span class="file-name">Choose the attachment ...</span>
   <% end %>
 
-  <%= link_to_remove_association "x", f, wrapper_class: "attachment" %>
+  <%= link_to_remove_association f, wrapper_class: "attachment" do %>
+    <span class="remove-outcome" style="margin: 0">Delete</span>
```
Comment (mborsare) [inline app/views/attachments/_fields.html.erb]: I will circle back and delete this inline style.

File: app/assets/stylesheets/components/_add-item.scss [hunk 2]
```diff
@@ -35,8 +35,36 @@
   transition: inherit;
 
   @at-root {
```
Comment (alexandra-aas) [inline app/assets/stylesheets/components/_add-item.scss]: Whoa... I've never seen this before

File: app/assets/stylesheets/components/_headline-poster.scss [hunk 3]
```diff
@@ -28,6 +28,16 @@
   color: tint($brown, 20%);
 }
 
+.headline-poster-faint {
+  @include anchor-color(tint($brown, 80%));
+  text-transform: lowercase;
+  font-weight: normal;
```
Comment (alexandra-aas) [inline app/assets/stylesheets/components/_headline-poster.scss]: 🔤

File: app/assets/stylesheets/components/_add-item.scss (hunk 2 above)
Comment (mborsare) [inline app/assets/stylesheets/components/_add-item.scss]: If you want to read more on it: http://sassbreak.com/getting-back-to-our-roots/

**Pull request merged**